test/src/module/command/archivePushTest.c

```diff
@@ -521,11 +521,36 @@ testRun(void)
             true, "check repo3 for WAL file");
 
         // -------------------------------------------------------------------------------------------------------------------------
-        TEST_TITLE("remove WAL from one repo and push again");
+        TEST_TITLE("write error on one repo but other repo succeeds");
 
         storageRemoveP(
             storageTest, strNewFmt("repo2/archive/test/11-1/0000000100000001/000000010000000100000002-%s", walBuffer2Sha1),
             .errorOnMissing = true);
+        storageRemoveP(
+            storageTest, strNewFmt("repo3/archive/test/11-1/0000000100000001/000000010000000100000002-%s", walBuffer2Sha1),
+            .errorOnMissing = true);
+
+        HRN_STORAGE_MODE(storageTest, "repo2/archive/test/11-1/0000000100000001", .mode = 0500);
+
+        TEST_ERROR(
+            cmdArchivePush(), CommandError,
+            "archive-push command encountered error(s):\n"
+            "repo2: [FileOpenError] unable to open file '/home/vagrant/test/test-0/repo2/archive/test/11-1/0000000100000001"
+                "/000000010000000100000002-755defa48a0a0872767b6dea49bdd3b64902f147' for write: [13] Permission denied");
```

Comment:
So, at this point, even though one repo succeeded, we don't indicate that because there was a hard error on the other repo?

Reply:
Yes -- any repos not in this list worked, but I think it add clutter to report that.